packages/teams-js/src/internal/globalVars.ts

```diff
@@ -3,10 +3,10 @@ export class GlobalVars {
   public static initializeCalled = false;
   public static initializeCompleted = false;
   public static additionalValidOrigins: string[] = [];
-  public static initializePromise: Promise<void>;
+  public static initializePromise: Promise<void> | null = null;
```

Comment:
> public static initializePromise: Promise<void> | null = null; [](http://example.com/codeflow?start=1&length=62)

Similar to the below, this variable was already getting a value of `undefined` because it wasn't being initialized. However, various parts of code elsewhere was setting this to `null` in certain cases. To reflect those existing `null` use cases, I updated the type to accept `null` (since it was already being used) and explicitly set the default value to `null` instead of `undefined` since the existing code conventions indicated that `null` was being used for the uninitialized case rather than `undefined`.